docker/server/Dockerfile

```diff
@@ -1,21 +1,23 @@
 # NOTE: The docker build context directory must be the root of the source tree.
 # NOTE: To build the release image, SQLFlow must be built into ./build
-FROM ubuntu:18.04
+FROM alpine:3.12
 
 # Choose faster mirrors for apt-get and pip
 COPY docker/dev/find_fastest_resources.sh /usr/local/bin/
-RUN /bin/bash -c 'source find_fastest_resources.sh \
-        && echo "Choose the fastest APT source ..." \
-        && choose_fastest_apt_source \
+RUN /bin/sh -c 'source find_fastest_resources.sh \
+        && echo "Choose the fastest Alpine source ..." \
+        && choose_fastest_alpine_source \
         && echo "Choose the fastest PIP source ..." \
         && choose_fastest_pip_source'
 
 # Install Python and JRE.  SQLFlow server requires Couler/Fluid to generate YAML
 # files, and Couler/Fluid depends on Python.  The external parsers are in Java.
-RUN apt-get -qq install -y \
-           python3-dev python3-pip openjdk-8-jre-headless > /dev/null \
-        && rm -rf /var/lib/apt/lists/* \
-        && apt-get -qq clean -y
+RUN apk update \
```

Comment:
Can put all `RUN` in one line to reduce the Docker image layers.

Reply:
Done, I put some relative RUN together